build/templates/session.py.mako

```diff
@@ -93,9 +93,9 @@ class _RepeatedCapabilities(object):
 
     def __getitem__(self, repeated_capability):
         '''Set/get properties or call methods with a repeated capability (i.e. channels)'''
-        rep_caps = _converters.convert_repeated_capabilities(repeated_capability, self._prefix)
+        rep_caps, rep_caps_list = _converters.convert_repeated_capabilities(repeated_capability, self._prefix)
 
-        return _SessionBase(${config['session_handle_parameter_name']}=self._session._${config['session_handle_parameter_name']}, repeated_capability=rep_caps, library=self._session._library, encoding=self._session._encoding, freeze_it=True)
+        return _SessionBase(${config['session_handle_parameter_name']}=self._session._${config['session_handle_parameter_name']}, repeated_capability=rep_caps, repeated_capability_list=rep_caps_list, library=self._session._library, encoding=self._session._encoding, freeze_it=True)
```

Comment:
If both rep_caps, rep_caps_list came from the same repeated_capability, why do you need to pass both to _SessionBase.__init__()?

Shouldn't you pass just one and let _SessionBase process it?

Reply:
So you pass `repeated_capability = 'ScriptTrigger0,ScriptTrigger1', rep_caps_list =  ['ScriptTrigger0', 'ScriptTrigger1']`? Still seems very redundant. Maybe I misunderstand, but there's no documentation in __init__().

Reply:
I was. I fixed the logic in `convert_repeated_capabilities()` and now it only returns the list.